.flake8

```diff
@@ -6,7 +6,6 @@
 # E402: module level import not at top of file
 # I100: Import statements are in the wrong order
 # I101: Imported names are in the wrong order. Should be
+max-line-length = 88
```

Comment:
I'd love for us to *not* have a max-line-length enforced by flake8. Perhaps we can copy https://github.com/jupyterhub/jupyterhub/blob/main/.flake8?

Reply:
This is so that flake8 is compatible with blacks autoformatting, which will default to 88. I was running off this suggestion https://github.com/2i2c-org/infrastructure/issues/938#issuecomment-1017609298

Reply:
@yuvipanda this is to be compliant with black, flake8 will not enforce anything harder - this is a relaxation I think.

Reply:
Unless this line means we are automatically ignoring `E501` errors irrespective of line-length?

https://github.com/2i2c-org/infrastructure/pull/939/files#diff-6951dbb399883798a226c1fb496fdb4183b1ab48865e75eddecf6ceb6cf46442R10

Reply:
Yeah hmm...

Well since we have a very relaxed config saying....
```
# Ignore style and complexity
# E: style errors
# W: style warnings
# C: complexity
ignore = E, C, W, ...
```

The max-line-length may be entirely irrelevant, while it could be relevant if we went with a minimal config to ensure it was compatible with black. This was the example they suggested in blacks doc.

```
[flake8]
max-line-length = 88
extend-ignore = E203
```

---

Since we now have a lot of ignored things making the config relaxed, I think the line length detail is irrelevant so +1 for removing it unless we tighten up the flake8 config.

Reply:
I think we should remove `max-line-length` and keep the other ignores we currently have. Will do that now.

Reply:
@sgibson91 I think you can also remove E402 since its included in E btw

Reply:
Done and done!

Reply:
Yay! Thank you for the explanation too :)